### PR TITLE
Bumps rust-g to `3.0.0` 

### LINKED ISSUE #76398: Flaky test organ_set_bonus_sanity: rustg returned illegally formatted string '1687996890' (code/modules/logging/log_holder.dm:256)
Flaky tests were detected in [this test run](https://github.com/tgstation/tgstation/actions/runs/5406953160/attempts/1). This means that there was a failure that was cleared when the tests were simply restarted.
Failures:
```
organ_set_bonus_sanity: [00:01:30] Runtime in stack_trace.dm,4: rustg returned illegally formatted string '1687996890' (code/modules/logging/log_holder.dm:256)
proc name:  stack trace (/proc/_stack_trace)
usr: *no key*/(Unknown)
usr.loc: (Test Room (126,126,15))
src: null
call stack:
stack trace("rustg returned illegally forma...", "code/modules/logging/log_holde...", 256)
/datum/log_holder (/datum/log_holder): human readable timestamp(3)
/datum/log_category/debug_mobt... (/datum/log_category/debug_mobtag): create entry("TAG: mob_3478 SPECIES: *no key...", null, null)
/datum/log_holder (/datum/log_holder): Log("debug-mobtag", "TAG: mob_3478 SPECIES: *no key...", null)
Unknown (/mob/living/carbon/human/consistent): log mob tag("TAG: mob_3478 SPECIES: *no key...", null)
Unknown (/mob/living/carbon/human/consistent): set species(/datum/species/human (/datum/species/human), 1, 0)
Unknown (/mob/living/carbon/human/consistent): set species(/datum/species/human (/datum/species/human), 1, 0)
Unknown (/mob/living/carbon/human/consistent): Initialize(0)
Atoms (/datum/controller/subsystem/atoms): InitAtom(Unknown (/mob/living/carbon/human/consistent), 0, /list (/list))
Unknown (/mob/living/carbon/human/consistent): New(0)
Unknown (/mob/living/carbon/human/consistent): New(the floor (126,126,15) (/turf/open/floor/iron))
/datum/unit_test/organ_set_bon... (/datum/unit_test/organ_set_bonus_sanity): allocate(/mob/living/carbon/human/consi... (/mob/living/carbon/human/consistent))
/datum/unit_test/organ_set_bon... (/datum/unit_test/organ_set_bonus_sanity): Run()
RunUnitTest(/datum/unit_test/organ_set_bon... (/datum/unit_test/organ_set_bonus_sanity), /list (/list))
RunUnitTests()
/datum/callback (/datum/callback): InvokeAsync() at stack_trace.dm:4
```

### LINKED ISSUE #76490: Flaky test on_mob_end_metabolize: rustg returned illegally formatted string '1688269905' (code/modules/logging/log_holder.dm:256)
Flaky tests were detected in [this test run](https://github.com/tgstation/tgstation/actions/runs/5434562453/attempts/1). This means that there was a failure that was cleared when the tests were simply restarted.
Failures:
```
on_mob_end_metabolize: [03:51:45] Runtime in stack_trace.dm,4: rustg returned illegally formatted string '1688269905' (code/modules/logging/log_holder.dm:256)
proc name:  stack trace (/proc/_stack_trace)
usr: *no key*/(John Doe)
usr.loc: (Test Room (126,126,13))
src: null
call stack:
stack trace("rustg returned illegally forma...", "code/modules/logging/log_holde...", 256)
/datum/log_holder (/datum/log_holder): human readable timestamp(3)
/datum/log_category/debug_mobt... (/datum/log_category/debug_mobtag): create entry("TAG: mob_2894 CREATED: *no key...", null, null)
/datum/log_holder (/datum/log_holder): Log("debug-mobtag", "TAG: mob_2894 CREATED: *no key...", null)
John Doe (/mob/living/carbon/human/consistent): log mob tag("TAG: mob_2894 CREATED: *no key...", null)
John Doe (/mob/living/carbon/human/consistent): Initialize(0)
John Doe (/mob/living/carbon/human/consistent): Initialize(0)
John Doe (/mob/living/carbon/human/consistent): Initialize(0)
John Doe (/mob/living/carbon/human/consistent): Initialize(0)
Atoms (/datum/controller/subsystem/atoms): InitAtom(John Doe (/mob/living/carbon/human/consistent), 0, /list (/list))
John Doe (/mob/living/carbon/human/consistent): New(0)
John Doe (/mob/living/carbon/human/consistent): New(the floor (126,126,13) (/turf/open/floor/iron))
/datum/unit_test/on_mob_end_me... (/datum/unit_test/on_mob_end_metabolize): allocate(/mob/living/carbon/human/consi... (/mob/living/carbon/human/consistent))
/datum/unit_test/on_mob_end_me... (/datum/unit_test/on_mob_end_metabolize): Run()
RunUnitTest(/datum/unit_test/on_mob_end_me... (/datum/unit_test/on_mob_end_metabolize), /list (/list))
RunUnitTests()
/datum/callback (/datum/callback): InvokeAsync() at stack_trace.dm:4
```

### LINKED ISSUE #76502: Flaky test create_and_destroy: rustg returned illegally formatted string '1688323734' (code/modules/logging/log_holder.dm:256)
Flaky tests were detected in [this test run](https://github.com/tgstation/tgstation/actions/runs/5437970194/attempts/1). This means that there was a failure that was cleared when the tests were simply restarted.
Failures:
```
create_and_destroy: [18:48:54] Runtime in stack_trace.dm,4: rustg returned illegally formatted string '1688323734' (code/modules/logging/log_holder.dm:256)
proc name:  stack trace (/proc/_stack_trace)
usr: *no key*/(Tatiana Driggers)
usr.loc: (Test Room (126,126,13))
src: null
call stack:
stack trace("rustg returned illegally forma...", "code/modules/logging/log_holde...", 256)
/datum/log_holder (/datum/log_holder): human readable timestamp(3)
/datum/log_category/debug_mobt... (/datum/log_category/debug_mobtag): create entry("TAG: mob_4630 SPECIES: *no key...", null, null)
/datum/log_holder (/datum/log_holder): Log("debug-mobtag", "TAG: mob_4630 SPECIES: *no key...", null)
Tatiana Driggers (/mob/living/carbon/human/species/skeleton): log mob tag("TAG: mob_4630 SPECIES: *no key...", null)
Tatiana Driggers (/mob/living/carbon/human/species/skeleton): set species(/datum/species/skeleton (/datum/species/skeleton), 1, 0)
Tatiana Driggers (/mob/living/carbon/human/species/skeleton): set species(/datum/species/skeleton (/datum/species/skeleton), 1, 0)
Tatiana Driggers (/mob/living/carbon/human/species/skeleton): set species(/datum/species/skeleton (/datum/species/skeleton), null, null)
Tatiana Driggers (/mob/living/carbon/human/species/skeleton): Initialize(0)
Atoms (/datum/controller/subsystem/atoms): InitAtom(Tatiana Driggers (/mob/living/carbon/human/species/skeleton), 0, /list (/list))
Tatiana Driggers (/mob/living/carbon/human/species/skeleton): New(0)
Tatiana Driggers (/mob/living/carbon/human/species/skeleton): New(the floor (126,126,13) (/turf/open/floor/iron))
/datum/unit_test/create_and_de... (/datum/unit_test/create_and_destroy): Run()
RunUnitTest(/datum/unit_test/create_and_de... (/datum/unit_test/create_and_destroy), /list (/list))
RunUnitTests()
/datum/callback (/datum/callback): InvokeAsync() at stack_trace.dm:4
```

### LINKED ISSUE #76514: Flaky test monkey_business: rustg returned illegally formatted string '1688376360' (code/modules/logging/log_holder.dm:256)
Flaky tests were detected in [this test run](https://github.com/tgstation/tgstation/actions/runs/5442755337/attempts/1). This means that there was a failure that was cleared when the tests were simply restarted.
Failures:
```
monkey_business: [09:26:00] Runtime in stack_trace.dm,4: rustg returned illegally formatted string '1688376360' (code/modules/logging/log_holder.dm:256)
proc name:  stack trace (/proc/_stack_trace)
usr: *no key*/(Unknown)
usr.loc: (Test Room (126,126,14))
src: null
call stack:
stack trace("rustg returned illegally forma...", "code/modules/logging/log_holde...", 256)
/datum/log_holder (/datum/log_holder): human readable timestamp(3)
/datum/log_category/debug_mobt... (/datum/log_category/debug_mobtag): create entry("TAG: mob_3877 SPECIES: *no key...", null, null)
/datum/log_holder (/datum/log_holder): Log("debug-mobtag", "TAG: mob_3877 SPECIES: *no key...", null)
Unknown (/mob/living/carbon/human/consistent): log mob tag("TAG: mob_3877 SPECIES: *no key...", null)
Unknown (/mob/living/carbon/human/consistent): set species(/datum/species/human (/datum/species/human), 1, 0)
Unknown (/mob/living/carbon/human/consistent): set species(/datum/species/human (/datum/species/human), 1, 0)
Unknown (/mob/living/carbon/human/consistent): Initialize(0)
Atoms (/datum/controller/subsystem/atoms): InitAtom(Unknown (/mob/living/carbon/human/consistent), 0, /list (/list))
Unknown (/mob/living/carbon/human/consistent): New(0)
Unknown (/mob/living/carbon/human/consistent): New(the floor (126,126,14) (/turf/open/floor/iron))
/datum/unit_test/monkey_busine... (/datum/unit_test/monkey_business): allocate(/mob/living/carbon/human/consi... (/mob/living/carbon/human/consistent), null)
/datum/unit_test/monkey_busine... (/datum/unit_test/monkey_business): Run()
RunUnitTest(/datum/unit_test/monkey_busine... (/datum/unit_test/monkey_business), /list (/list))
RunUnitTests()
/datum/callback (/datum/callback): InvokeAsync() at stack_trace.dm:4
```

### LINKED ISSUE #76549: Flaky test greyscale_item_icon_states: rustg returned illegally formatted string '1688495129' (code/modules/logging/log_holder.dm:256)
Flaky tests were detected in [this test run](https://github.com/tgstation/tgstation/actions/runs/5457510366/attempts/1). This means that there was a failure that was cleared when the tests were simply restarted.
Failures:
```
greyscale_item_icon_states: [18:25:29] Runtime in stack_trace.dm,4: rustg returned illegally formatted string '1688495129' (code/modules/logging/log_holder.dm:256)
proc name:  stack trace (/proc/_stack_trace)
src: null
call stack:
stack trace("rustg returned illegally forma...", "code/modules/logging/log_holde...", 256)
/datum/log_holder (/datum/log_holder): human readable timestamp(3)
/datum/log_category/debug_runt... (/datum/log_category/debug_runtime): create entry("::group::/datum/unit_test/grey...", null, null)
/datum/log_holder (/datum/log_holder): Log("runtime", "::group::/datum/unit_test/grey...", null)
log world("::group::/datum/unit_test/grey...", null)
RunUnitTest(/datum/unit_test/greyscale_ite... (/datum/unit_test/greyscale_item_icon_states), /list (/list))
RunUnitTests()
/datum/callback (/datum/callback): InvokeAsync() at stack_trace.dm:4
```

### LINKED ISSUE #76798: Flaky test spawn_mobs: rustg returned illegally formatted string '1689188061' (code/modules/logging/log_holder.dm:256)
Flaky tests were detected in [this test run](https://github.com/tgstation/tgstation/actions/runs/5535298990/attempts/1). This means that there was a failure that was cleared when the tests were simply restarted.
Failures:
```
spawn_mobs: [18:54:21] Runtime in stack_trace.dm,4: rustg returned illegally formatted string '1689188061' (code/modules/logging/log_holder.dm:256)
proc name:  stack trace (/proc/_stack_trace)
usr: *no key*/(gutlunch)
usr.loc: (Test Room (126,126,14))
src: null
call stack:
stack trace("rustg returned illegally forma...", "code/modules/logging/log_holde...", 256)
/datum/log_holder (/datum/log_holder): human readable timestamp(3)
/datum/log_category/debug_mobt... (/datum/log_category/debug_mobtag): create entry("TAG: mob_3666 CREATED: *no key...", null, null)
/datum/log_holder (/datum/log_holder): Log("debug-mobtag", "TAG: mob_3666 CREATED: *no key...", null)
the gutlunch (/mob/living/simple_animal/hostile/asteroid/gutlunch): log mob tag("TAG: mob_3666 CREATED: *no key...", null)
the gutlunch (/mob/living/simple_animal/hostile/asteroid/gutlunch): Initialize(0)
the gutlunch (/mob/living/simple_animal/hostile/asteroid/gutlunch): Initialize(0)
the gutlunch (/mob/living/simple_animal/hostile/asteroid/gutlunch): Initialize(0)
the gutlunch (/mob/living/simple_animal/hostile/asteroid/gutlunch): Initialize(0)
the gutlunch (/mob/living/simple_animal/hostile/asteroid/gutlunch): Initialize(0)
the gutlunch (/mob/living/simple_animal/hostile/asteroid/gutlunch): Initialize(0)
Atoms (/datum/controller/subsystem/atoms): InitAtom(the gutlunch (/mob/living/simple_animal/hostile/asteroid/gutlunch), 0, /list (/list))
the gutlunch (/mob/living/simple_animal/hostile/asteroid/gutlunch): New(0)
the gutlunch (/mob/living/simple_animal/hostile/asteroid/gutlunch): New(the floor (126,126,14) (/turf/open/floor/iron))
/datum/unit_test/spawn_mobs (/datum/unit_test/spawn_mobs): allocate(/mob/living/simple_animal/host... (/mob/living/simple_animal/hostile/asteroid/gutlunch))
/datum/unit_test/spawn_mobs (/datum/unit_test/spawn_mobs): Run()
RunUnitTest(/datum/unit_test/spawn_mobs (/datum/unit_test/spawn_mobs), /list (/list))
RunUnitTests()
/datum/callback (/datum/callback): InvokeAsync() at stack_trace.dm:4
```

### LINKED ISSUE #76921: Flaky test strange_reagent: rustg returned illegally formatted string '1689594546' (code/modules/logging/log_holder.dm:256)
Flaky tests were detected in [this test run](https://github.com/tgstation/tgstation/actions/runs/5575152170/attempts/1). This means that there was a failure that was cleared when the tests were simply restarted.
Failures:
```
strange_reagent: [11:49:06] Runtime in stack_trace.dm,4: rustg returned illegally formatted string '1689594546' (code/modules/logging/log_holder.dm:256)
proc name:  stack trace (/proc/_stack_trace)
usr: *no key*/(nurse spider)
usr.loc: (Test Room (126,126,15))
src: null
call stack:
stack trace("rustg returned illegally forma...", "code/modules/logging/log_holde...", 256)
/datum/log_holder (/datum/log_holder): human readable timestamp(3)
/datum/log_category/debug_mobt... (/datum/log_category/debug_mobtag): create entry("TAG: mob_4258 CREATED: *no key...", null, null)
/datum/log_holder (/datum/log_holder): Log("debug-mobtag", "TAG: mob_4258 CREATED: *no key...", null)
the nurse spider (/mob/living/basic/giant_spider/nurse): log mob tag("TAG: mob_4258 CREATED: *no key...", null)
the nurse spider (/mob/living/basic/giant_spider/nurse): Initialize(0)
the nurse spider (/mob/living/basic/giant_spider/nurse): Initialize(0)
the nurse spider (/mob/living/basic/giant_spider/nurse): Initialize(0)
the nurse spider (/mob/living/basic/giant_spider/nurse): Initialize(0)
the nurse spider (/mob/living/basic/giant_spider/nurse): Initialize(0)
Atoms (/datum/controller/subsystem/atoms): InitAtom(the nurse spider (/mob/living/basic/giant_spider/nurse), 0, /list (/list))
the nurse spider (/mob/living/basic/giant_spider/nurse): New(0)
the nurse spider (/mob/living/basic/giant_spider/nurse): New(the floor (126,126,15) (/turf/open/floor/iron))
/datum/unit_test/strange_reage... (/datum/unit_test/strange_reagent): allocate(/mob/living/basic/giant_spider... (/mob/living/basic/giant_spider/nurse))
/datum/unit_test/strange_reage... (/datum/unit_test/strange_reagent): allocate new target(/mob/living/basic/giant_spider... (/mob/living/basic/giant_spider/nurse))
/datum/unit_test/strange_reage... (/datum/unit_test/strange_reagent): Run()
RunUnitTest(/datum/unit_test/strange_reage... (/datum/unit_test/strange_reagent), /list (/list))
RunUnitTests()
/datum/callback (/datum/callback): InvokeAsync() at stack_trace.dm:4
```

### LINKED ISSUE #76934: Flaky test create_and_destroy: rustg returned illegally formatted string '1689653927' (code/modules/logging/log_holder.dm:256)
Flaky tests were detected in [this test run](https://github.com/tgstation/tgstation/actions/runs/5583174400/attempts/1). This means that there was a failure that was cleared when the tests were simply restarted.
Failures:
```
create_and_destroy: [04:18:47] Runtime in stack_trace.dm,4: rustg returned illegally formatted string '1689653927' (code/modules/logging/log_holder.dm:256)
proc name:  stack trace (/proc/_stack_trace)
usr: *no key*/(Erica Kimple)
usr.loc: (Test Room (126,126,13))
src: null
call stack:
stack trace("rustg returned illegally forma...", "code/modules/logging/log_holde...", 256)
/datum/log_holder (/datum/log_holder): human readable timestamp(3)
/datum/log_category/debug_mobt... (/datum/log_category/debug_mobtag): create entry("TAG: mob_4691 CREATED: *no key...", null, null)
/datum/log_holder (/datum/log_holder): Log("debug-mobtag", "TAG: mob_4691 CREATED: *no key...", null)
Erica Kimple (/mob/living/carbon/human/monkeybrain): log mob tag("TAG: mob_4691 CREATED: *no key...", null)
Erica Kimple (/mob/living/carbon/human/monkeybrain): Initialize(0)
Erica Kimple (/mob/living/carbon/human/monkeybrain): Initialize(0)
Erica Kimple (/mob/living/carbon/human/monkeybrain): Initialize(0)
Erica Kimple (/mob/living/carbon/human/monkeybrain): Initialize(0)
Atoms (/datum/controller/subsystem/atoms): InitAtom(Erica Kimple (/mob/living/carbon/human/monkeybrain), 0, /list (/list))
Erica Kimple (/mob/living/carbon/human/monkeybrain): New(0)
Erica Kimple (/mob/living/carbon/human/monkeybrain): New(the floor (126,126,13) (/turf/open/floor/iron))
/datum/unit_test/create_and_de... (/datum/unit_test/create_and_destroy): Run()
RunUnitTest(/datum/unit_test/create_and_de... (/datum/unit_test/create_and_destroy), /list (/list))
RunUnitTests()
/datum/callback (/datum/callback): InvokeAsync() at stack_trace.dm:4
```

### LINKED ISSUE #76950: Flaky test create_and_destroy: rustg returned illegally formatted string '1689726984' (code/modules/logging/log_holder.dm:256)
Flaky tests were detected in [this test run](https://github.com/tgstation/tgstation/actions/runs/5593505367/attempts/1). This means that there was a failure that was cleared when the tests were simply restarted.
Failures:
```
create_and_destroy: [00:36:24] Runtime in stack_trace.dm,4: rustg returned illegally formatted string '1689726984' (code/modules/logging/log_holder.dm:256)
proc name:  stack trace (/proc/_stack_trace)
usr: *no key*/(guard spider)
usr.loc: (Test Room (126,126,13))
src: null
call stack:
stack trace("rustg returned illegally forma...", "code/modules/logging/log_holde...", 256)
/datum/log_holder (/datum/log_holder): human readable timestamp(3)
/datum/log_category/debug_mobt... (/datum/log_category/debug_mobtag): create entry("TAG: mob_5099 CREATED: *no key...", null, null)
/datum/log_holder (/datum/log_holder): Log("debug-mobtag", "TAG: mob_5099 CREATED: *no key...", null)
the guard spider (/mob/living/basic/giant_spider/guard): log mob tag("TAG: mob_5099 CREATED: *no key...", null)
the guard spider (/mob/living/basic/giant_spider/guard): Initialize(0)
the guard spider (/mob/living/basic/giant_spider/guard): Initialize(0)
the guard spider (/mob/living/basic/giant_spider/guard): Initialize(0)
the guard spider (/mob/living/basic/giant_spider/guard): Initialize(0)
the guard spider (/mob/living/basic/giant_spider/guard): Initialize(0)
Atoms (/datum/controller/subsystem/atoms): InitAtom(the guard spider (/mob/living/basic/giant_spider/guard), 0, /list (/list))
the guard spider (/mob/living/basic/giant_spider/guard): New(0)
the guard spider (/mob/living/basic/giant_spider/guard): New(the floor (126,126,13) (/turf/open/floor/iron))
/datum/unit_test/create_and_de... (/datum/unit_test/create_and_destroy): Run()
RunUnitTest(/datum/unit_test/create_and_de... (/datum/unit_test/create_and_destroy), /list (/list))
RunUnitTests()
/datum/callback (/datum/callback): InvokeAsync() at stack_trace.dm:4
```

### PR DESCRIPTION
**Pull request by san7890**
## About The Pull Request

You can read more about what's introduced in the last four versions here:

https://github.com/tgstation/rust-g/releases

The immediate effects for us are as follows:
Closes #76549
Closes #76514
Closes #76502
Closes #76490
Closes #76398
Closes #76798
Closes #76921
Closes #76934
Closes #76950

the updates to install rust_g.sh were to cut down on the copypasta code and prevent desyncs.

:cl:
code: The currently operating rust-g version on a live server is posted to places like the runtime.log, in the same place where the revision information and any applicable test merges already were.
/:cl:

### CODE REVIEW

**Comment by san7890** [comment]
image

why has ci been stuck for 5 hours

**Comment by san7890** [comment]
image

what the glorg

**Comment by san7890** [comment]
no goddammit what the hell you infernal machine

**Comment by san7890** [comment]
i hope to god we don't have to go back to compiling rust-g every compile what is going on here

**Comment by san7890** [comment]
> ldd (Ubuntu GLIBC 2.31-0ubuntu9.9) 2.31

whart

**Comment by san7890** [comment]
alright absolutely none of my changes has fixed this thing just absolutely bricking and failing. the more shit i add here is probably pretty simple on the rust-g side of affairs but that will have to wait a day as i sleep and just do other stuff. just baffling.

**Comment by san7890** [comment]
this pr is probably stunlocked until rust-g can figure out the external dependencies situation or we just pave over the situation with installing more packages? unsure because my brain is frying like an egg in this heat and i only remember half of the conversation

**Comment by san7890** [comment]
SQL and everything (appears to) works on campbell after hijacking TGS via ce331dda19a333f8cea88581a258e1ca00e80f1d

**Comment by Sealed101** [comment]
another one #76950

**Comment by san7890** [comment]
i have recieved no reports of catastrophic failure with any SQL functions (bans and playtime tracking being the most visible that could have broken) on either Sybil (running Windows) or Campbell (running Linux) so I believe this PR should be good to go:tm: when https://github.com/tgstation/rust-g/pull/144 is merged and `3.0.0` is released.

**Comment by ZeWaka** [comment]
released

**Comment by MrStonedOne** [comment]
Test merge this on campbell to make sure linux works before merging.

**Comment by Cyberboss** [comment]
> SQL and everything (appears to) works on campbell after hijacking TGS via [ce331dd](https://github.com/tgstation/tgstation/commit/ce331dda19a333f8cea88581a258e1ca00e80f1d)

You know TGS scripts are manually uploaded and not pulled from the repo right?

**Comment by san7890** [comment]
I could’ve sworn I had it break by updating the scripts repo-side

**Comment by san7890** [comment]
alright any test merge reports past round-210972 for Sybil and round-210967 for Campbell have the latest commit of this branch

**Comment by MrStonedOne** [comment]
you should confirm the new rustg version is getting used on campbell someway.

**Comment by san7890** [comment]
Campbell:

image

The proc called (`rustg_unix_timestamp()`) affirms that it is on the 3.0.0 release of rust-g, as any previous versions will return a number like `1.69031e+009`

**Review by MrsTonedOne — APPROVED** [review]

File: tools/ci/install_rust_g.sh
Comment (Cyberboss) [inline tools/ci/install_rust_g.sh:7]: These two lines by themselves do nothing

File: tools/tgs_scripts/InstallDeps.sh
Comment (Cyberboss) [inline tools/tgs_scripts/InstallDeps.sh:20]: Bro, use `curl`

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by ZeWaka** [comment]
Why are you still installing libssl in InstallDeps? Is it for something besides rust-g?

**Comment by san7890** [comment]
unsure, odds are that it is related to rust-g since we build it using TGS in the PreCompile.sh step. I'll remove it and see if the Test TGS Docker step fails

File: tools/tgs_scripts/InstallDeps.sh
Comment (Cyberboss) [inline tools/tgs_scripts/InstallDeps.sh:24]: Holdup, these are deps of auxtools (and therefore auxlua) as well.

Idk how, but they are.

Comment (Cyberboss) [reply tools/tgs_scripts/InstallDeps.sh:24]: CC @willox @Mothblocks

Comment (Cyberboss) [reply tools/tgs_scripts/InstallDeps.sh:24]: IIRC it's DMASM using an old ass spacemandmm/dreammaker version

Comment (san7890) [reply tools/tgs_scripts/InstallDeps.sh:24]: Does Test TGS Docker not do auxtools/auxlua stuff? Might be worth looking into in the future so we wouldn't immediately shrek ourselves on accident like we were just about to here.

I'll revert it in the meantime

Comment (Cyberboss) [reply tools/tgs_scripts/InstallDeps.sh]: As long as it's not in `PreCompile.sh`, it doesn't

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**
